### DISCUSSION TITLE
Support request for Plant monitor (BLE devices) via Qingping BLE gateway

### DISCUSSION THREAD

**Discussion by DjAnu** [merdok/homebridge-miot#537] [Ideas]
Is it possible to see devices that are connected through BLE gateway ? I am able to see my Qingping BLE gateway but I don't see any devices (plant sensors) that are connected through the gateway. Is it possible to expose those to Homekit ?

I see there are discussions about them and seems like they are added as BLE devices. However I have them setup the BLE gateway so I can access them over wifi & cloud anywhere.

Thanks.

**Comment by merdok** [comment]
Hi,

yes it is possible.
You should specify the token and ip of the gateway, but the model and deviceId needs to be that of the plant monitor.
For each BLE device you need to make a separate config entry.

**Reply by DjAnu** [reply]
That's great. How do I get token of the gateway. I don't see it anywhere in MiHome App. Also where would I see device ID/model of plant monitors. MiHome App doesn't show any of these. Thanks for your help.

**Reply by merdok** [reply]
You can use the cli commands to connect to your cloud account and get a list of all devices including tokens, models and ip numbers

https://github.com/merdok/homebridge-miot#cli

**Reply by DjAnu** [reply]
Awesome Thanks for your help. I am now able to see and add plant monitors. However I am not able to add them as tiles in my Home App. They appear as an accessory with all 4 properties but there is no option to add them to home view. If I add them to a room then properties show up as climate but combined in 1 bar. What do I need to do to add them to home ?

**Reply by merdok** [reply]
This is now a home app issue. Not sure how to do it there, you could try to split them into single tiles instead.

**Reply by DjAnu** [reply]
Problem is the way it shows up in the home
app I don’t see an option to split it either. See the screenshotss

**Reply by merdok** [reply]
Yeah I mean this is how the home app shows it, not sure if that can be changed.

**Comment by DjAnu** [comment]
![IMG_6395](https://github.com/merdok/homebridge-miot/assets/5419410/fb3383ae-9b75-4473-bbab-cbc4ae5048d0)
![IMG_6394](https://github.com/merdok/homebridge-miot/assets/5419410/99210b42-53ff-4d74-936c-6cf8403f5bf6)

**Comment by DjAnu** [comment]
Ah, I was so excited that I could add Plant monitors on my home App. Is
there any way to customize how the device info is being sent to Homekit ?

**Reply by merdok** [reply]
You can play around with the propertyMonitor although I am not sure if that changes something.
